cmflib/cli/utils.py

```diff
@@ -1,4 +1,71 @@
+import subprocess
+import os
+
+
 def fix_subparsers(subparsers):
     subparsers.required = True
     subparsers.dest = "cmd"
 
+
+def create_cmf_config(file_name: str, cmf_server_ip: str):
+    with open(file_name, "w") as file:
+         file.write(f"cmf.server.ip={cmf_server_ip}")
+
+def read_cmf_config(file_name: str):
+    with open(file_name, "r") as file:
+         return file.read()
+
+def git_exists():
+    try:
+        output = subprocess.check_output(["git", "version"]).decode("ascii")
+        return output
+    except FileNotFoundError:
+        return "\033[1;31mERROR:\033[1;m git is not installed!! Install git."
+
+def find_root(file_name: str):
+
+    msg = "'cmf' is not configured.\nExecute 'cmf init' command."
+    root_dir = os.path.realpath(os.getcwd())
+    while True:
+        file_path = os.path.join(root_dir, file_name)
+        if os.path.exists(file_path):
+            return root_dir
+        if os.path.ismount(root_dir):
+            return msg
+        parent = os.path.abspath(os.path.join(root_dir, os.pardir))
+        if parent == root_dir:
```

Comment:
what is the scenario for this if statement ?

Reply:
It was not needed. Removed.